xl_auth/settings.py

```diff
@@ -35,6 +35,8 @@ class Config(object):
     OAUTH2_PROVIDER_TOKEN_EXPIRES_IN = int(
         os.environ.get('OAUTH2_PROVIDER_TOKEN_EXPIRES_IN', 3600))
     MAX_ALLOWED_ACTIVE_PASSWORD_RESETS = 2
+    FAILED_LOGIN_TIMEFRAME_MINUTES = 60
```

Comment:
@mxtthias This becomes custom setting no. 2 and 3. What do you think about adopting the convention used by other modules? I.e.

`BABEL_DEFAULT_LOCALE = ...` = comes from Flask-Babel plugin
`EMAIL_TIMEOUT = ...` = coming from Flask-Mail plugin
`SQLALCHEMY_TRACK_MODIFICATIONS = ...` = comes from Flask-SQLAlchemy

Proposing a refactoring like so:
```python
    XL_AUTH_MAX_ACTIVE_PASSWORD_RESETS = 2
    XL_AUTH_MAX_FAILED_LAGIN_ATTEMPTS = 7
    XL_AUTH_FAILED_LOGIN_EXPIRES_IN = 3600
```

In my subtle way, I'm also proposing consistent use of SI base unit for time. When using standard units, we can stop using suffixes like `..._MINUTES`, `..._HOURS`, `..._SECONDS`, `..._DAYS`, `..._WEEKS`. This is a common Python convention, in my experience. See surrounding 3rd-party settings; *EMAIL_TIMEOUT = 5*,  *OAUTH2_PROVIDER_TOKEN_EXPIRES_IN = 3600*.

Another thing I'm subtly conveying with the renaming proposal is consistent terminology. For example, in our models we already have the concept that 1) something **expires_at** a certain time and 2) **expires_in** a timeframe of *(now() - N)* seconds. I think the logic in *FailedLoginAttempts* is very much the same; after a number of seconds the application will no longer take the login failure into consideration. So let's not have a different word for it. 

@mxtthias What do you think? Too much [foolish consistency](https://www.python.org/dev/peps/pep-0008/#a-foolish-consistency-is-the-hobgoblin-of-little-minds)?